rosbag2_transport/test/rosbag2_transport/test_play_seek.cpp

```diff
@@ -212,7 +212,7 @@ TEST_P(RosBag2PlaySeekTestFixture, seek_forward) {
   await_received_messages.get();
 
   auto replayed_topic1 = sub_->get_received_messages<test_msgs::msg::BasicTypes>("/topic1");
-  EXPECT_THAT(replayed_topic1, SizeIs(expected_number_of_messages));
+  ASSERT_THAT(replayed_topic1, SizeIs(expected_number_of_messages));
```

Comment:
Since we're accessing `replayed_topic1` after this, we should assert its size. Same thing with the other instances below.